src/woocommerce.js

```diff
@@ -0,0 +1,9 @@
+/*--------------------------------------------------------------
+# Emma - Webpack Admin Entrypoint
```

Comment:
This comment should be `Emma - Webpack WooCommerce Entrypoint`

Reply:
handled in d8bc22e61094f0ed3da2e85fa890fbfb6b504fd6